src/main/java/in/projecteka/datanotificationsubscription/DataNotificationSubscriptionConfiguration.java

```diff
@@ -331,4 +310,42 @@ public UserServiceClient userServiceClient(
                 authorizationHeader);
     }
 
+    @Bean
+    public ReactorClientHttpConnector reactorClientHttpConnector() {
+        HttpClient httpClient = null;
+        try {
+            SslContext sslContext = SslContextBuilder
+                    .forClient()
+                    .trustManager(InsecureTrustManagerFactory.INSTANCE)
+                    .build();
+            httpClient = HttpClient.create().secure(t -> t.sslContext(sslContext));
+        } catch (SSLException e) {
+            e.printStackTrace();
+        }
+        return new ReactorClientHttpConnector(Objects.requireNonNull(httpClient));
+    }
+
+    @Bean("customBuilder")
+    public WebClient.Builder webClient(
+            @Qualifier("subscriptionHttpConnector") final ClientHttpConnector clientHttpConnector,
+            ObjectMapper objectMapper) {
+        return WebClient
+                .builder()
+                .exchangeStrategies(exchangeStrategies(objectMapper))
+                .clientConnector(clientHttpConnector)
+                .codecs(configurer -> configurer
```

Comment:
We should not need this! The whole `codecs` part was needed because we wanted to QR code on CM as image.